packages/components/stamp/src/stamp.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
-import InsetSquish from '@commercetools-uikit/spacings-inset-squish';
```

Comment:
Can we remove it from the package dependencies?

Reply:
Thanks. Done https://github.com/commercetools/ui-kit/pull/1638/commits/331a4b4f288b4ef8d1f693681d43fec176e4359f